welcomescreen.py

```diff
@@ -79,31 +83,54 @@ def run(self, game):
 
             game.screen.fill(BLACK)
             # Show Welcome screen
-            if ruleflag == 0:
-                game.screen.blit(self.welcomescreen, (350, 0))
+            if (ruleflag == 0):
+                game.screen.blit(self.welcomescreen, (game.background_x, 0))
+                # Check if start button is hovered
                 if self.play_button.get_rect(
-                        center=(530 + 60, 300 + 60)).collidepoint(
-                            mos_x, mos_y):
-                    game.screen.blit(pygame.transform.scale(
-                        self.play_button, (124, 124)), (530 - 2, 300 - 2))
+                   center=((game.middle_of_screen_x),
+                           (game.screen_height // 2))).collidepoint(
+                               mos_x, mos_y):
+                    self.play_button_rescaled = pygame.transform.scale(
+                        self.play_button, (int(self.play_dimensions * 1.1),
+                                           int(self.play_dimensions * 1.1)))
+                    game.screen.blit(
+                        self.play_button_rescaled, (
+                            game.middle_of_screen_x - (
+                                self.play_dimensions) * 1.1 // 2,
+                            game.screen_height // 2 - (
+                                self.play_dimensions) * 1.1 // 2))
```

Comment:
```python3
game.screen.blit(
    self.play_button_rescaled,
    game.middle_of_screen_x \
    - (self.play_dimensions * 1.1) // 2,
    game.screen_height // 2 \
    - (self.play_dimensions * 1.1) // 2
)
```

Seems more readable as we know were the blit starts and ends.

For the grouping of the parentheses, `self.play_dimensions` being grouped by itself doesn't sit right with me so I'm guessing you multiply and then divide the product of that.